common/azure_auth_test.go

```diff
@@ -272,6 +272,23 @@ func TestAzureAuth_configureWithClientSecret(t *testing.T) {
 	assert.Len(t, zi.Zones, 3)
 }
 
+func TestAzureEnvironment_WithAzureManagementEndpoint(t *testing.T) {
```

Comment:
This test is not testing anything new :) please remove

Reply:
done